projects/fsastorefrontlib/src/cms-components/questionnaire-carousel/questionnaire-carousel.component.html

```diff
@@ -92,15 +92,3 @@ <h5 class="notice-text text-center p-4 m-0">
     }}</a>
   </h5>
 </ng-template>
-
-<ng-template #noResults>
```

Comment:
why is this removed?

Reply:
There is a check above, if there is a combination of answers that doesn't provide any result, noResults template should be displayed

Reply:
Yes, but this is duplicated. There is same template ref above it. 🙂

Reply:
Aha ok. Probably issue with merge conflict :)